packages/react-charting/docs/TestPlans/HeatMapChart/ComponentTests.md

```diff
@@ -0,0 +1,30 @@
+# Heatmap Chart - Component test plan
+
+## Snapshot tests
+
+| Test case                                                                    | Validation                            | Tool used |
+| ---------------------------------------------------------------------------- | ------------------------------------- | --------- |
+| With only data, domainValuesForColorScale and rangeValuesForColorScale props | Should render heatmap chart correctly | Enzyme    |
+| When data is not present for some group                                      | Should render heatmap chart correctly | Enzyme    |
+| When hideLegend prop is set to true                                          | Should render heatmap chart correctly | Enzyme    |
+| When yAxisTickFormat prop is set                                             | Should render heatmap chart correctly | Enzyme    |
```

Comment:
Can you add the expected behavior for each validation entry.

Reply:
done